extensions/wikia/PortableInfoboxBuilder/services/PortableInfoboxBuilderService.class.php

```diff
@@ -210,9 +219,13 @@ protected function getFormattedMarkup( $xml, $formatted ) {
 					// we close the current group and append it to the infobox dom...
 					$infoboxDom->appendChild( $this->importNodeToDom( $infoboxDom, $currentGroupDom ) );
 
+					$collapsible = $this->isCollapsible($currentChildNode);
+
+					// this attribute is not supported in header tag
+					$childNodeDom->removeAttribute('section_collapsible');
+
 					// and initialize a new group
-					$currentGroupDom = $this->createGroupDom();
-					$currentGroupDom->appendChild( $currentGroupDom->ownerDocument->importNode( $childNodeDom, true ) );
+					$currentGroupDom = $this->createGroupDom($childNodeDom, $collapsible);
```

Comment:
This is intentional here - since childNodeDom is mutated, doing it in separate function would decrease the readabiilty of this code